docs/tutorials/build-an-outdoor-rover.md

```diff
@@ -0,0 +1,304 @@
+
+1. One from the pin you added labeled **G** to any GND terminal on the pi
+2. One from the pin you added labeled **P** to any available GPIO pin on the pi
+3. One from terminal labeled **DIRExternal potentiometer** to any available GPIO pin on the pi
+
+Finally, you'll want to connect the wires from the 12V to the 5v pi power converter to the battery.
+
+At this point, you can plug in any extra components to the Pi (GPS and camera, if you are using them).
+If they are USB-based as per the parts list, this is easy.
+
+## Bring your robot to life
+
+### Viam config overview
+
+We've now got all the hardware and wiring set up, so it is time to configure your robot to be controlled with software.
+You might be bracing yourself, as this part is usually much harder than what you just did.
+Thankfully, the Viam platform makes it relatively painless.
+
+First, let's install [viam-server](/installation/prepare/rpi-setup/) on your pi.
+It should only take a few minutes and once you're done, you are ready to configure your robot.
+
+The Viam platform represents both individual hardware pieces and logical groupings of hardware as [components](/components/).
+Higher-level control is surfaced with [services](/services/).
+For this project, we'll configure:
+
+- One [board](/components/board/) component representing our Raspberry Pi
+- Two [motor](/components/motor/) components
+- One [base](/components/base/) component, referencing the two motors
+
+Configuring these components will allow you to control your rover through the internet both manually and programmatically.
+Later, we can do more interesting things by configuring:
+
+- A [camera](/components/camera/) component (optional but recommended)
+- A [movement sensor](/components/movement-sensor/) component, which in this case represents GPS (optional)
+- A [vision](/services/vision/) service, which allows us to use machine learning models to detect various things in the robot environment
+
+### Setting up our Boxbot config using the Viam app
+
+#### Board component
+
+<div class="td-max-width-on-larger-screens">
+  <img src="../img/outdoor-rover-boxbot/board-setup.png" alt="Board setup." title="Board setup." />
+</div>
+
+In the [Viam app](http://app.viam.com/), navigate to **CONFIG** within the robot you created previously, and add a new component of type "board", model "pi".
+You can name it whatever you want, but you'll reference this name later.
+We'll call it "local" in this example.
+
+#### Motor component
+
+Next, create another component of type "motor", model "gpio".
+Call it "left" to represent the motor on the left side of your rover.
+Now we'll need to add some attributes.
+At minimum, we'll need to reference the board component we created in the last step by name, and the GPIO pins we connected from this motor to our pi.
+Other attributes (Max RPM, PWM Freq) are [important for motor control](/components/motor/gpio#brushless-dc-motor) and can vary based on your specific hub motors.
```

Comment:
i feel like add some attributes you used if its necessary for this tutorial, saying "they are important but can vary based on the motor" makes it confusing